posts/month-in-mathlib-oct-2021.md

```diff
@@ -0,0 +1,86 @@
+---
+author: 'Mathlib community'
+category: 'month-in-mathlib'
+date: 2021-11-01 06:38:50+00:00
+description: ''
+has_math: true
+link: ''
+slug: month-in-mathlib-oct-2021
+tags: ''
+title: This month in mathlib (Oct 2021)
+type: text
+---
+
+This post summarizes some of the activity that happened in mathlib in October.
+
+## Highlighted PRs
+
+* Recently, there has been a surge in activity in the algebro-geometric part of the library.
+  After the definition of schemes entered mathlib, the development stalled for a while,
+  because the Liquid Tensor Experiment took up a lot of energy and attention.
+  But new contributors showed up, and are pushing the library forward.
+  Two highlights this month include:
+  - [PR #9416](https://github.com/leanprover-community/mathlib/pull/9416) Global sections of the spectrum of a ring are isomorphic to that ring.
+  - [PR #9861](https://github.com/leanprover-community/mathlib/pull/9861) Closed points in the prime spectrum correspond to maximal ideals.
+
+* [PR #9323](https://github.com/leanprover-community/mathlib/pull/9323) :: feat(probability_theory/density): define probability density functions
+* [PR #9378](https://github.com/leanprover-community/mathlib/pull/9378) :: feat(measure_theory/function/conditional_expectation): conditional expectation on real functions equal Radon-Nikodym derivative
+* [PR #9469](https://github.com/leanprover-community/mathlib/pull/9469) :: feat(probability_theory/notation): add notations for expected value, conditional expectation
+
+* [PR #9229](https://github.com/leanprover-community/mathlib/pull/9229) :: feat(ring_theory/algebraic_independent): algebraic independence
+* [PR #9377](https://github.com/leanprover-community/mathlib/pull/9377) :: feat(ring_theory/algebraic_independent): Existence of transcendence bases and rings are algebraic over transcendence basis
+
+* [PR #9462](https://github.com/leanprover-community/mathlib/pull/9462) :: feat(measure_theory/covering/besicovitch): the Besicovitch covering theorem
+* [PR #9461](https://github.com/leanprover-community/mathlib/pull/9461) :: feat(measure_theory/covering/besicovitch_vector_space): vector spaces satisfy the assumption of Besicovitch covering theorem
+* [PR #9576](https://github.com/leanprover-community/mathlib/pull/9576) :: feat(measure_theory/covering/besicovitch): the measurable Besicovitch covering theorem
+* [PR #9679](https://github.com/leanprover-community/mathlib/pull/9679) :: feat(measure_theory/covering/besicovitch): remove measurability assumptions
+* [PR #9680](https://github.com/leanprover-community/mathlib/pull/9680) :: feat(measure_theory/covering/vitali): Vitali covering theorems
+
```

Comment:
The Vitali and Besicovitch covering theorems ensure that, from a covering of a (not necessarily measurable) set `s` by nice enough sets (e.g. balls), one can extract a disjoint subfamily covering almost all `s`. They are instrumental in forthcoming results on differentation of measures. This series of PRs proves the Vitali and Besicovitch covering theorems (with the optimal constant for the Besicovitch covering theorem, following [Füredi and Loeb, On the best constant for the Besicovitch covering theorem][furedi-loeb1994]).